core/pom.xml

```diff
@@ -61,6 +61,11 @@
       <groupId>io.jsonwebtoken</groupId>
       <artifactId>jjwt</artifactId>
     </dependency>
+    <dependency>
+      <!-- required as a (missing) transient dependency for JJWT -->
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
```

Comment:
I would rather put this into a profile which is activated by the java version:
```
  <profiles>
    <profile>
      <id>java-10-deps</id>
      <activation>
        <jdk>10</jdk>
      </activation>
      <dependencies>
        <dependency>
          <groupId>javax.xml.bind</groupId>
          <artifactId>jaxb-api</artifactId>
        </dependency>
      </dependencies>
    </profile>
  </profiles>
```
This way the jar gets only included in the Spring Boot jar when we build on Java 10. WDYT?

Reply:
I don't think it hurts the Java 8 deployment. And it would spare as the extra profile. So I would vote for keeping it simple and not adding any conditionals unless absolutely necessary.